Dockerfile

```diff
@@ -1,18 +1,82 @@
-FROM continuumio/miniconda3
+FROM nvidia/cuda:11.2.0-cudnn8-devel-ubuntu20.04 AS build
```

Comment:
See what [RasterVision has done](https://github.com/azavea/raster-vision/blob/master/Dockerfile#L1), maybe the cuda version could be inputted by the user as an variable.

Reply:
> ... {CUDA_VERSION}-cudnn7 ...

This is probably specific to their set up.  I'm not sure it's wise to allow the CUDA_VERSION be passed as an argument.  What if we input 11.2 and there were no cudnn7/8 Ubuntu images for that CUDA version ?

Also Nvidia has a [CUDA compatibility matrix](https://docs.nvidia.com/deeplearning/cudnn/support-matrix/index.html) which shows that CUDA versions have to go along with the GPU host's CUDA Driver version.  I'd rather stick with CUDA and cudnn versions that we know works on our systems.

Reply:
Indeed, I ran into the "What if we input 11.2 and there were no cudnn7/8 Ubuntu images for that CUDA version" problem. It is a big downside.